.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
please revert, don't make unrelated changes like this in PRs

Reply:
Done in another PR